File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService', function ($http) {

    this.findAll=function () {
        return $http.get('../brand/findAll.do');
    }

    this.findPage=function (pageNum, pageSize) {

        return $http.post('../brand/findByPage.do?pageNum='+pageNum+'&pageSize='+pageSize);
    }

    this.searchByPage=function (pageNum, pageSize, searchEn) {

        return $http.post('../brand/searchByPage.do?pageNum='+pageNum+'&pageSize='+pageSize, searchEn);
    }

    this.update=function (poj) {

        return $http.post('../brand/update.do', poj);
    }

    this.findOne=function (id) {

        return $http.get('../brand/findOne.do?id='+id);
    }

    this.delete=function (ids) {

        return $http.post('../brand/delete.do?ids='+ids);
    }

    this.selectOptionsList=function () {

        return $http.get('../brand/selectOptionsList.do');
    }


});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface BrandService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbBrand> findAll();

	public PageResult findByPage(int pageNum, int pageSize);

	/*
		按照条件查询并分页
	 */
	public PageResult findByPage(TbBrand brand, int pageNum, int pageSize);

	public void update(TbBrand brand);

	public TbBrand findOne(Long id);

	public void delete(Long ids[]);

	List<Map> selectOptionsList();
	
}
